setup.py

```diff
@@ -1,28 +1,35 @@
-import codecs, os
-from setuptools import setup, find_packages
+import codecs
+import os
+
+from setuptools import find_packages, setup
+
 
 def read(fname):
     return codecs.open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+
 setup(
-    name='relaycard',
-    version='0.1',
```

Comment:
As soon as we merge this, we should bump the version to 1.0 as we introduce a major breaking change by dropping Python 2 (finally).